tensorflow/tensorflow.bzl

```diff
@@ -27,79 +27,79 @@ load(
 
 # List of proto files for android builds
 def tf_android_core_proto_sources(core_proto_sources_relative):
-  return ["//tensorflow/core:" + p
+  return [str(Label("//tensorflow/core:" + p))
```

Comment:
Could you use the clean_dep() wrapper defined here?
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tensorflow.bzl#L49

I think it's a bit more readable. Same everywhere.